### PR TITLE
Split views: config options for issue #44

### PR DESCRIPTION
**Pull request by bert-w**
I've added a small layer of config to customize the view settings. This is a non-breaking patch.

### CODE REVIEW

File: src/controllers/LogViewerController.php [hunk 1]
```diff
@@ -27,10 +32,33 @@ public function index()
 
         $logs = LaravelLogViewer::all();
 
-        return View::make('laravel-log-viewer::log', [
+        return View::make(self::cfgHas('view.log') ? self::cfg('view.log') : 'laravel-log-viewer::log', [
```
Comment (mikemand) [inline src/controllers/LogViewerController.php]: The `config` helper has an optional second parameter that is used as default if the config key doesn't exist:

`config(self::$configFile . '.view.log', 'laravel-log-viewer::log')`

**Comment by rap2hpoutre** [comment]
Ok thank you. I think addition to the `readme` will be a bit confusing for most users. I want to keep the `readme` as simple as possible. What do you think of only adding this to a wiki?

**Comment by Bertuzb** [comment]
where do they find the wiki? Personally I would add it to the readme on the very bottom, and stress it is an "optional" config.

As long as people know the feature exists :)

**Comment by maarten00** [comment]
Should definately be in the readme, as that is where most users would be looking for 'basic' configuration options like this. 

Perhaps the 'Optional' part should be in the title as 
`Config (optional)`

Other then that +1 for this feature!

**Comment by rap2hpoutre** [comment]
2 years old PR, close it. Sorry for the time you spent and my lack of answer. Feel free to tell me if you feel I've done something wrong.

**Pull request closed without merging**